maths/prime_factorization_fast.py

```diff
@@ -0,0 +1,61 @@
+from math import ceil, sqrt
+from __future__ import annotations
```

Comment:
use `ruff . --fix` to fix all ruff issues.

Reply:
Resolved [Here](https://github.com/TheAlgorithms/Python/pull/8920/commits/2590a893f828a5b8e7462c1aae1d28f53263098b)